inference-engine/ie_bridges/python/sample/speech_sample/speech_sample.py

```diff
@@ -235,6 +248,24 @@ def main():
             if args.reference:
                 compare_with_reference(results[blob_name][key], references[blob_name][key])
 
+            if args.performance_counter:
+                pc = perf_counters[i]
+                total_cycles = int(pc["1.1 Total scoring time in HW"]["real_time"])
+                stall_cycles = int(pc["1.2 Stall scoring time in HW"]["real_time"])
+                active_cycles = total_cycles - stall_cycles
+                frequency = 10**6
+                if args.arch == "CORE":
+                    frequency *= 400
+                else:
+                    frequency *= 200
+                total_inference_time = total_cycles/frequency
+                active_time = active_cycles/frequency
+                stall_time = stall_cycles/frequency
+                print("\nPerformance Statistics of GNA Hardware")
+                print("   Total Inference Time: " + str(total_inference_time * 1000) + " ms")
+                print("   Active Time: " + str(active_time * 1000) + " ms") 
+                print("   Stall Time: " + str(stall_time * 1000) + " ms\n")
+
```

Comment:
It works only on harware GNA,  on software it always shows 0.0ms.
![image](https://user-images.githubusercontent.com/68621725/125771749-ebb6ffb4-94ee-400b-af22-bb06caeb47e5.png)

@dorloff Do we need to get a per-layer report as in C++ version?

Reply:
Yes, it's expected that it's 0.0 on SW emulation mode.
I think we can have a per-layer report, but don't need to do it in this PR.